src/vs/workbench/contrib/terminal/browser/terminalQuickFixBuiltinActions.ts

```diff
@@ -122,13 +121,12 @@ export function gitCreatePr(openerService: IOpenerService): ITerminalQuickFixOpt
 			if (!command) {
 				return;
 			}
-			const branch = matchResult?.outputMatch?.[1];
-			const link = matchResult?.outputMatch?.[2];
-			if (!branch || !link) {
+			const link = matchResult?.outputMatch?.[1];
+			if (!link) {
 				return;
 			}
 			const actions: IAction[] = [];
-			const label = localize("terminal.gitCreatePr", "Create PR");
+			const label = localize("terminal.openLink", "Open link: {0}", link);
```

Comment:
This is a bit vague - what about `Open PR link on GitHub`?

Reply:
Personally I think the URL is more transparent/useful than, especially early on when there's less trust in the feature; I'm scared of more bugs like https://github.com/microsoft/vscode/issues/162074 slipping in. Shall we keep it as open link for now refine in future iterations? Best of both worlds work be the new quick fix widget with the ability to show the URL/more info as well. Showing what free port in a details part would be nice too

Reply:
Yes that works for me